samples/local_attestation/common/attestation.cpp

```diff
@@ -91,25 +108,32 @@ bool Attestation::attest_local_report(
 
     TRACE_ENCLAVE("report_size = %ld", report_size);
 
+    verifier_result = oe_verifier_initialize();
+    if (verifier_result != OE_OK)
+    {
+        TRACE_ENCLAVE("oe_verifier_initialize failed.");
+        goto exit;
+    }
+
     // 1)  Validate the report's trustworthiness
     // Verify the report to ensure its authenticity.
-    result = oe_verify_report(local_report, report_size, &parsed_report);
+    result = oe_verify_evidence(&selected_format, local_report, report_size, NULL, 0, NULL, 0, &claims, &claims_length);
```

Comment:
Pass in NULL as the first parameter of oe_verify_evidence(). The format id is embedded in the report. No need to specify here.

Reply:
The format id is not embedded in the report because we specified NULL as the second parameter in oe_get_evidence().

Reply:
I see that you are using evidence without format id embedded. A selected format is sent from attester to verifier. oe_attester_select_format() is for picking a format from an array of format. If there is only one format to pick from, that's not a good demonstration of the function.

I'd suggestion remove the use of selected_format and assume both ends know this is for local attestation.

Reply:
@yentsanglee Currently we must still include the format id in the call to oe_verify_evidence, because when using the embedded format id, _oe_get_target_info_internal() fails because it cannot parse the new header format correctly. @yufjiang can provide more information on this issue as well.

Reply:
@kevinxu118, I opened the issue https://github.com/openenclave/openenclave/issues/3322 to track the identified gap here. Given that issue, you can keep your existing way to get and verify the evidence.